docs/hub/models-gated.md

```diff
@@ -1,31 +1,74 @@
 # Gated models
 
-To give model creators more control over how their models are used, the Hub allows users to enable **User Access requests** through a model's **Settings** tab.
-Enabling this setting requires users to agree to share their contact information and accept the model owners' terms and conditions in order to access the model.
-The contact information is stored in a database, and model owners are able to download a copy of the user access report.
+To give more control over how models are used, the Hub allows model authors to enable **Access requests** for their models. When enabled, users have to agree to share their contact information (username and email address) with the model authors to access the model files. A model with Access requests enabled is called a **Gated model**. Access requests are always granted to individual users and not to entire organization.
 
-Note that to download a gated model you'll need to be authenticated. You can log in by installing the [`huggingface_hub` client library](https://huggingface.co/docs/huggingface_hub/index) and running the following in your terminal:
+## Manage gated models as a model author
 
-```bash
-huggingface-cli login
-```
+### Configure access control
 
-If you're using a Jupyter or Colaboratory notebook, log in with the following Python command
+To enable Access requests, go to the model settings page. By default the model is not gated. Click on "Enable Access request" on the top-right corner.
 
-```python
-from huggingface_hub import notebook_login
-notebook_login()
-```
+![](https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-disabled.png)
```

Comment:
Please also add darkmode images

Reply:
Thanks for forbidding me to be lazy :smile: Done in https://github.com/huggingface/hub-docs/pull/1128/commits/91c17ee8a0f8d23e8bd48c21b8da98a55aba4ee7.